pkg/watermark/processor/processor_manager_test.go

```diff
@@ -19,11 +19,11 @@ package processor
 import (
```

Comment:
Changes to this file reverted a previous goroutine leak fix. Please apply the fix in https://github.com/numaproj/numaflow/pull/774

Reply:
may be got removed while resolving conflicts, will add that back.